src/PointwiseFunctions/GeneralRelativity/Tags.hpp

```diff
@@ -105,6 +105,12 @@ template <size_t Dim, typename Frame, typename DataType>
 struct TraceSpatialChristoffelSecondKind : db::SimpleTag {
   using type = tnsr::I<DataType, Dim, Frame>;
 };
+/// Contraction of the first two indices of the spatial Christoffel symbols:
+/// \f$\Gamma^i_{ij}\f$. Useful for covariant divergences.
+template <size_t Dim, typename Frame, typename DataType>
+struct SpatialChristoffelSecondKindContracted : db::SimpleTag {
```

Comment:
@markscheel @kidder Is this the most sensible name for Gamma_i, or is there already a different standardized name in the KST papers, SpEC, or elsewhere?

Reply:
Since the Christoffel symbols of the second kind are the standard ones nearly always used, I would suggest to just call them 'Christoffel' for brevity.   In the fewer cases where Gamma_{ijk} is needed, those can be called 'ChristoffelFirstKind'.